### PR TITLE
Rob - starting PR convo for terminology for Elements

### PR DESCRIPTION
**Pull request by huckocs**
I figured we could try a PR rather than a call, lets see how we get on

### CODE REVIEW

File: common/terminology.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#h2 Pattern 
+<p>A high level solution to a common design problem. May use multiple components put together in different ways. Generally a basis for solving a design problem, not a prescriotive solution</p>
```
Comment (alexkilgour) [inline common/terminology.md]: prescriptive

**Comment by alexkilgour** [comment]
makes sense to me from a fed pov

File: common/terminology.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+<p>Examples: tabs, video controls, checkout steps 
+
+#h2 Component 
+<p>Specific chunks of the UI that can be put together to compose a page. Each component is prescriptive and has a specific role</p>
```
Comment (sonniesedge) [inline common/terminology.md]: Punctuation.

File: common/terminology.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+<p>Examples: card, footer, banner, header
+
+#h2 Package
+<p>A block of code representing a component or some global code that can be installed in a product application through NPM</p>
+<p>Examples: article view, Nature CSS context, Global CSS utilities, card
```
Comment (sonniesedge) [inline common/terminology.md]: Rob, your lack of closing tags is killing me. 😂

**Comment by sonniesedge** [comment]
Actually, I've just realised this is a markdown file and doesn't need the HTML tags in there at all.

File: common/terminology.md (hunk 3 above)
Comment (huckocs) [reply common/terminology.md]: Just testing your spotting skillz 😆

**Comment by jpw** [comment]
Not sure if/where you are expecting this to appear on the Elements site...

**Comment by huckocs** [comment]
@jpw Nowhere at the moment, this is more a PR placeholder. We should think about how stuff like this fits into the IA too. As we found from the GDS talk on Patterns Day, turns out a content strategist is really bloody critical :/

**Comment by jpw** [comment]
LGTM but `tabs` would be a component, not sure if (also?) a pattern...

**Pull request closed without merging**